### ISSUE TITLE
Feature: Option to toggle visibility of "Cash Pop-ups"

### ISSUE THREAD

**Issue by Derpercoaster** [OpenLoco/OpenLoco#1273]
**Feature**
Whenever you make a transaction or a vehicle makes a profit, the amount of money in question appears as floating text above the location of the transaction. My suggestion is to add an option in the options menu to toggie the visibility of this, as it can become annoying when it covers large portions of the map, esepcially when you are building a lot of things in paused mode.

**Describe alternatives you've considered**
It could also be an option in the view tab (the menu that allows you to toggle height marks, invisible objects/roads, etc)

**Comment by LeftofZen** [comment]
Just wondering, if this were to be implemented, would you also silence the audio effect for the money effect?

**Comment by Derpercoaster** [comment]
> Just wondering, if this were to be implemented, would you also silence the audio for the money effect?

I suppose it could be optional, but it would make sense to leave the audio on, so you'd know if a vehicle is making money or not.

**Comment by duncanspumpkin** [comment]
One thing to remember is if this is implemented is that it should probably only be done in the drawing engine as otherwise it will modify state which will cause desyncs in network games. This would lead to a perhaps unexpected popup of money entities when switching it from off to on.

**Comment by LeftofZen** [comment]
Agreed, this would be a drawing toggle only, probably around `paintRedGreenCurrencyEntity()` in PaintMiscEntity.cpp. Was just trying to suss out the scope, this is probably a 'good first issue' since the paint code is already reverse-engineered here.
